docs/README.md

```diff
@@ -0,0 +1,349 @@
+
+<h1 id="devcycle-test-harness-proxy">DevCycle Test Harness - Proxy v1.0.0</h1>
+
+# Overview
+The SDK proxy server is responsible for wrapping the SDK under test with an HTTP interface that can be used to call any of its public-facing methods with an arbitrary set of arguments,
+and returning the response to the caller (which will be the harness server). It will also allow configuration options to be passed which override the base API URL the SDK makes requests to.
+Setting that URL to the test harness server’s address allows the harness to mock the backend APIs to return data back (or to trigger errors).
+
+### About `entityType`
+To check that the SDK methods are returning a correct type of an instance, they will all contain an `entityType` in their response. `enityType` can be one of five values:
+  `Variable`, `User`, `Feature`, `Object`, `Void`
+
+<h1 id="devcycle-test-harness-proxy-client">Client</h1>
+
+SDK Client
+
+<a id="opIdcreateClient"></a>
+
+`POST /client`
+
+*Create a DVC client instance*
+
+> Body parameter
+
+```json
+{
+  "clientId": "string",
+  "sdkKey": "string",
+  "options": {},
+  "enableCloudBucketing": true
+}
+```
+
+<h3 id="createclient-parameters">Parameters</h3>
+
+|Name|In|Type|Required|Description|
+|---|---|---|---|---|
+|`clientId`|body|string|true|A unique ID that will be used to reference the client once it has been created|
+|`sdkKey`|body|string|false|A unique key used to initialize the DVC SDK|
+|`options`|body|object|false|An object of launch parameters. See [DevCycle Docs](https://docs.devcycle.com/docs/sdk/server-side-sdks/node#initialization-options) for a full list of options|
+|`enableCloudBucketing`|body|boolean|false|An option to use the cloud bucketing version of the SDK|
+
+> Example responses
+
+> Created
+
+```json
+{
+  "message": "Success"
+}
+```
+
+```json
+{
+  "message": "Missing environment key! Call initialize with a valid environment key"
+}
+```
+
+> 400 Response
+
+```json
+{
+  "error": "Invalid request: missing clientId"
+}
+```
+
+<h3 id="createclient-responses">Responses</h3>
+
+|Status|Meaning|Description|
+|---|---|---|
+|201|[Created](https://tools.ietf.org/html/rfc7231#section-6.3.2)|Created|
+|400|[Bad Request](https://tools.ietf.org/html/rfc7231#section-6.5.1)|Bad Request|
+
+<h3 id="createclient-responseschema">Response Schema</h3>
+
+Status Code **201**
+
+|Name|Type|Required|Description|
+|---|---|---|---|
+|`» message`|string|true|A message indicating the result of initializing an SDK client|
+
+Status Code **400**
+
+|Name|Type|Required|Description|
+|---|---|---|---|
+|`» error`|string|true|An error message indicating what went wrong with the request|
+
+### Response Headers
+
+|Status|Header|Type|Format|Description|
+|---|---|---|---|---|
+|201|Location|string||Location of the Client instance on the proxy server|
+
+<a id="opIdcommand"></a>
+
+`POST /client/{clientId}/command`
```

Comment:
Is this URL correct? Pretty sure it currently does not have the "command" at the end

Reply:
I thought we wanted to add "command" at the end. You brought it up on slack so I assumed we'd make the change. I'll remove it for now if that's no longer the plan

Reply:
ok i'll make a ticket to change the endpoint